crates/oxc_linter/src/rules/unicorn/no_null.rs

```diff
@@ -303,5 +305,45 @@ fn test() {
         ("Object.create(bar, null)", None),
     ];
 
-    Tester::new(NoNull::NAME, pass, fail).test_and_snapshot();
+    let fix = vec![
+        ("let x = null;", "let x;", None),
+        ("let x = null as any;", "let x = undefined as any;", None),
+        ("if (foo == null) {}", "if (foo == undefined) {}", None),
+        ("if (foo != null) {}", "if (foo != undefined) {}", None),
+        ("if (foo == null) {}", "if (foo == undefined) {}", Some(check_strict_equality(true))),
+        // FIXME
+        (
+            "if (foo === null || foo === undefined) {}",
```

Comment:
i wonder whether when the `null` literal is in a comparison (while condition/if condition/switch condition/binary expression (equality)) we just shouldn't fix it - it's probably gonna break people's code

Reply:
I'm thinking something similar. That, or we make it a suggestion.